projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-file-view/case-file-view-field.component.spec.ts

```diff
@@ -233,4 +242,25 @@ describe('CaseFileViewFieldComponent', () => {
     const errorMessagesFromElements = listElements.map(item => item.nativeElement.textContent);
     expect(component.errorMessages).toEqual(errorMessagesFromElements);
   });
+
+  it('should enable icp with test config', () => {
+    fixture.detectChanges();
+    component.isIcpEnabled();
+    expect(component.icpEnabled).toBeTruthy();
+  });
+
+  it('should disable icp with disabled config', () => {
+    mockabstractConfig.getIcpEnable.and.returnValue(false);
+    fixture.detectChanges();
+    const callIcpEnabled = component.isIcpEnabled();
```

Comment:
This isn't quite right either. You are not really testing what you want to test, which is that when the config contains false, and you create the component, icp is disabled.